Firebase/Auth/Source/FIRAuth.m

```diff
@@ -1092,25 +1094,25 @@ - (nullable NSString *)languageCode {
 
 - (void)setLanguageCode:(nullable NSString *)languageCode {
   dispatch_sync(FIRAuthGlobalWorkQueue(), ^{
-    _requestConfiguration.languageCode = [languageCode copy];
+    self.requestConfiguration.languageCode = [languageCode copy];
```

Comment:
I'd be consistent and use iVar instead of getter inside the class. Same for all below. (Unless there is already precedence otherwise)

Reply:
Done, thanks.